scripts/spack

```diff
@@ -165,7 +166,15 @@ spack_install_dependencies ()
 	then
 		for dependency in $(spack_get_dependencies "install-${mode}")
 		do
+			if test ${dependency} = 'mysql'
+			then
+				(
+					cd "${SPACK_DIR}"
+					./bin/spack install mysql +client_only build_type=Release cxxstd=11
+				)
+			else
 			spack_install "${dependency}"
+			fi
```

Comment:
This should also use `spack_install`. Also, indentation is wrong in the else branch.

Reply:
the problem with mysql is, that it requires "spaces" within the install command to specify the arguments.
The indention is wrong, yes.